gofmt.go

```diff
@@ -108,8 +108,6 @@ func processFile(filename string, in io.Reader, out io.Writer, stdin bool) error
 		}
 	}
 
-	ast.SortImports(fileSet, file)
```

Comment:
not sure if I should have left it there and copy it, or move it. Seems like both works and I can't find an edgecase where it doesn't.